libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -59,4 +57,10 @@ export class BookSearchComponent implements OnInit {
       this.store.dispatch(clearSearch());
     }
   }
+  ngOnDestroy() {
+    if (this.books) {
+      this.books.unsubscribe();
+      this.store.dispatch(clearSearch());
+    }
+  }
```

Comment:
Unsubscribed on ngDestroy() which resolved memory leak issues in the application.